tests/test_files.py

```diff
@@ -104,11 +104,13 @@ def test_create_invld_windows(self):
         with pytest.raises(CuckooOperationalError):
             Folders.create("Z:\\invalid\\directory")
 
+    @pytest.mark.xfail(os.geteuid() == 0, reason="nektos act runs tests as root")
     def test_delete_invld(self):
         """Test deletion of a folder we can't access."""
         dirpath = tempfile.mkdtemp()
 
         os.chmod(dirpath, 0)
+        # Deleting the directory will succeed if the tests are run as root.
```

Comment:
When run by nektos/act and the effective userid is 0, we're able to delete the directory even when the permissions are 0.  So it won't raise the expected CuckooOperationalError.